docs/_data/portland-2025-config.yaml

```diff
@@ -155,11 +155,12 @@ about:
   job_fair_room: Sponsor area
 
 cfp:
-  url: ""                                         # Open October 6 (Mon) -> 9 weeks to submit
-  ends: "January 21, 2025"                        # (Mon) 1 week, to review. It'll take a little longer to announce
+  url: "https://pretalx.com/wtd-portland-2025/cfp" # Open October 6 (Mon) -> 9 weeks to submit
+  ends: "January 21, 2025"                         # (Mon) 1 week, to review. It'll take a little longer to announce
   notification: "February 04, 2025"                # (Mon) 8 weeks to write and prepare
+  info_by: "April 21, 2025"                        # (Mon) 2 weeks before for info for emcee
```

Comment:
We need info much before that for social media posting, etc.

Reply:
Yep, this is reverted in the CFP page, and I'll remove from here as well.

Reply:
I also need to update the emails to go hand in hand.